### ISSUE TITLE
2.1 Keyboard Accessible

### ISSUE THREAD

**Issue by DonnaDuiker** [alkem-io/client-web#8371]
# Background
After going through the WCAG 2.1 checklist, creating issues for all A and AA points that still need work. 1 issue for each guideline.

This is guideline 2.1 Keyboard Accessible

# Acceptance Criteria

- [ ] 2.1.1 Keyboard: All functionality is available via keyboard. (Should be Checked, tooltips?)
- [ ] 2.1.2 No Keyboard Trap: Keyboard focus is not trapped in any element. (Should be Checked)

# Context

[Excel file](https://stichtingcherrytwist.sharepoint.com/:x:/s/Product/EawxIsWzBaRHt8K2E1tUwmsBzd93mWRRiImOP2M0muxujQ?e=eYV2Qr)

[W3C website](https://www.w3.org/TR/WCAG21/)
